src/components/Loading/Loading.css

```diff
@@ -0,0 +1,129 @@
+/**
+ * ----------------------------------------
+ * Début règle animation heartbeat
+ * ----------------------------------------
+ */
+  @keyframes heartbeat {
+    from {
+      -webkit-transform: scale(1);
```

Comment:
React uses a tool called autoprefixer which adds vendor prefixes automatically. You don't need to write `-webkit-`